main.go

```diff
@@ -19,11 +19,13 @@ import (
 )
 
 var (
-	allowedHosts argumentList
-	imaginaryURL string
-	listenPort   int64
-	bucketRate   float64
-	bucketSize   int64
+	allowedHosts            argumentList
+	allowedImaginaryParams  string
+	allowedImaginaryActions string
```

Comment:
Shouldn't `allowedImaginaryParams` and `allowedImaginaryActions` be `argumentList`s ?
Or maybe another list type that accepts comma-separated values and decode them into a `[]string` ?